src/index.test.ts

```
@@ -0,0 +1,137 @@
+  describe('getChainId', () => {
+    it('should be defined and callable', () => {
+      expect(getChainId).toBeDefined();
+      expect(typeof getChainId).toBe('function');
+    });
+
+    it('should return chain ID for a given network name', () => {
+      const result = getChainId('avalanche_mainnet');
+      expect(result).toBe(43114); // Based on what was shown in the JSON
+    });
+
+    it('should return chain ID for an alias', () => {
```

ESLint

[error] Replace `'should·return·chain·ID·for·an·alias'` with `"should·return·chain·ID·for·an·alias"`

(prettier/prettier)